admin-console/src/main/webapp/src/pages/BundlePage/BehaviorGroupTable.tsx

```diff
@@ -53,25 +54,35 @@ export const BehaviorGroupsTable: React.FunctionComponent<BundlePageProps> = (pr
     const handleSubmit = React.useCallback((systemBehaviorGroup) => {
         setShowModal(false);
         const mutate = newBehaviorGroup.mutate;
+        const updateActionsMutate = updateBehaviorActions.mutate;
         mutate({
             id: systemBehaviorGroup.id,
             displayName: systemBehaviorGroup.displayName ?? '',
-            actions: systemBehaviorGroup.actions,
-            bundleId: systemBehaviorGroup.bundleId
-        }).then(getBehaviorGroups.query);
-
-    }, [ getBehaviorGroups.query, newBehaviorGroup.mutate ]);
+            bundleId: props.bundleId
+        })
+        .then(response => {
+            if (response.payload?.status === 200 && (response.payload.value.id || systemBehaviorGroup.id)) {
+                return updateActionsMutate({
+                    behaviorGroupId: response.payload.value.id ?? systemBehaviorGroup.id,
+                    body: [
+                        {
+                            ignore_preferences: false,
+                            only_admins: systemBehaviorGroup.actions === 'email-admin'
+                        }
+                    ]
+                });
+            }
+        })
```

Comment:
Above block is the one that takes care of updating the actions -
Since this is shared with the create/update we check wether the id comes from the response or the request.